src/components/Button/Button.js

```diff
@@ -0,0 +1,30 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import './Button.css'
+
+class Button extends Component{
+    handleClick = () => {
+        this.props.handleClick(this.props.name)
+    }
+    render() {
+        return (
+            <div onClick={this.handleClick} className={'Button'} style={{backgroundColor: this.props.color, width: this.props.wide ? '49.7%' : '24.7%'}}>
+                {this.props.name}
+            </div>
+        )
+    }
+}
+
+Button.propTypes = {
+    name: PropTypes.string.isRequired,
+    color: PropTypes.string,
+    wide: PropTypes.bool,
+    handleClick: PropTypes.func
+}
+
+Button.defaultProps = {
```

Comment:
Why did we need to use `defaultProps` here when we aren't making `color` a required prop?

Reply:
I needed to set the default values for some props that aren't required. props like color and width

Reply:
While yeah it appears without the `defaultProps` the colors and the width are affected, there was a better way to do it. The thing about React, just like Ruby on Rails, we technically can do anything as long as things work. But the question we should always ask is "is this the best way to implement this using this framework/libray?". So I suggest the following:

1. Instead of packing all your styles inline you should have just created `classNames` to handle `color` and `wide` props for you. 

2.To avoid having to set `defaultProps` for your `color` prop you could have:

```
<div style={backgroundColor: props.color || "#e89353"} >....
```

3. For your `wide` prop you could also have avoided using `defaultProps` by simply passing `true` or `false` values from your `ButtonPanel` component as `wide` prop:

`buttonPanel.js`:
```
<Button
   handleClick={this.props.clickHandler} name={'8'} color={'#e0e0e0'}
   wide={false} (or true depending on whether the button should be wide or not)
/>
```